src/data/faqEntry/index.tsx

```diff
@@ -1,28 +1,34 @@
 import * as R from 'ramda'
-import {
-  Locale,
-  extractSingleTranslation,
-  TranslationCollection,
-} from '../i18n'
+import { Locale, TranslationCollection, filterByLocale } from '../i18n'
 import { useLocale } from '../../utils/hooks'
 import data from './faq.json'
+import { Head } from '../../components/Head'
 
 export interface FAQPage {
   headline: string
   subheadline: string
-  metaTitle: string
   faqEntries: FAQEntry[]
+  head: Head
 }
 
-const faq: TranslationCollection<FAQPage> = data
+export type FAQPageOnDisk = {
+  id: string
+  head: TranslationCollection<Head>
+} & TranslationCollection<{
+  headline: string
+  subheadline: string
+  faqEntries: FAQEntry[]
+}>
+
+const faq: FAQPageOnDisk = data
 
 export interface FAQEntry {
   question: string
   answer: string
 }
 
-const getFAQPage = (locale: Locale): FAQPage =>
-  extractSingleTranslation<FAQPage>(locale)(faq)
+export const getFAQPage = (locale: Locale): FAQPage =>
+  filterByLocale(locale)(faq)
```

Comment:
I change to the filter here since we don't have only one big `TranslationCollection` anymore.